app/reducers/taps.js

```diff
@@ -179,6 +186,13 @@ const defaultState = {
       app_secret: '',
       start_date: ''
     }
+  },
+  'tap-s3-csv': {
+    fieldValues: {
+      bucket: '',
+      start_date: '',
+      tables: ''
```

Comment:
Should tables default to `[]` instead of `''`?

Reply:
Converting to a string before saving to store since the config expects it to be a string.